account_fiscalyear_close/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2018 Jarsa Sistemas, S.A. de C.V.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Account Fiscal Year Close",
+    "summary": "Account Fiscal Year Close",
+    "version": "10.0.1.0.0",
+    "category": "Hidden",
+    "website": "https://www.jarsa.com.mx/",
+    "author": "Jarsa Sistemas",
+    "license": "AGPL-3",
+    "installable": True,
```

Comment:
This key is not necessary, please remove it

Reply:
ok